<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintBrushSettingsWidget</class>
 <widget class="QWidget" name="PaintBrushSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>133</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelRadius">
         <property name="text">
          <string>&amp;Radius:</string>
         </property>
         <property name="buddy">
          <cstring>sliderRadius</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRadiusValue">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="sliderRadius">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAntialias">
     <property name="text">
      <string>&amp;Anti-aliasing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
